tests/opae-c/test_props_c.cpp

```diff
@@ -30,6 +30,7 @@
 #include "mock/opae_fixtures.h"
 #include "props.h"
 
+#include<bits/stdc++.h>
```

Comment:
Can this be replaced with #include <array>

Reply:
fixed